released/packages/coq-dijkstra/coq-dijkstra.0.1.0/opam

```diff
@@ -0,0 +1,32 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A Verified Implementation of Dijkstra's Algorithm"
+maintainer: ["Masayuki Mizuno <mizuno@wantedly.com>"]
+authors: ["Masayuki Mizuno <mizuno@wantedly.com>"]
+license: "MIT"
+homepage: "https://github.com/fetburner/coq-dijkstra"
+bug-reports: "https://github.com/fetburner/coq-dijkstra/issues"
+depends: [
+  "dune" {>= "2.5"}
+  "coq" {>= "8.8"}
+  "coq-mathcomp-ssreflect" {>= "1.11"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/fetburner/coq-dijkstra.git"
```

Comment:
There should be some tags too, as explained into the README, like:
```
tags: [
  "keyword:cool"
  "keyword:stuff"
  "category:Some/Category"
  "date:1992-12-22"
  "logpath:SomePrefix"
]
```
(at least the `date` and `logpath`)

Reply:
Thank you for your suggestion. I have supplemented tag informations.